src/views/Transfer/Transfer.html

```diff
@@ -62,13 +71,14 @@
       :selectedTab="selectedTab"
       :accounts="accounts"
       :selectedAccount="selectedAccount"
+      :selectedCurrency="selectedCurrency"
     >
     </FormComponent>
 
     <v-row>
       <v-col>
         <span v-if="$route.query.tab != 'receive'"
-          >Fee: 0.1 TFT</span
+          >Fee: 0.1 {{selectedCurrency}}</span
```

Comment:
We have to check if its always 0.1

Reply:
For now it is 0.1 on all currencies, made issue for later https://github.com/threefoldtech/3bot_wallet/issues/79